### DISCUSSION TITLE
Date field with current date does not fill value in a form

### DISCUSSION THREAD

**Discussion by indigoram89** [laravel/nova-issues#3715] [Q&A]
- Laravel Version: 8.77.1
- Nova Version: 3.30.0
- PHP Version: 8.0.8
- Operating System and Version: MacOS 11.6
- Browser type and version: Safari 15.0

### Detailed steps to reproduce the issue on a fresh Nova installation:
1. Create a model with timestamp column.
2. Add the column to $casts as 'datetime'.
3. Create record with current day in the timestamp column.
4. Open edit page of this resource in Laravel Nova.
5. The Date field does not have a value.
6. 
![img-2022-01-26-10-42-49](https://user-images.githubusercontent.com/85135210/151122857-aa267910-3677-4ad2-96ea-2c4915e96821.png)

<img width="527" alt="Screenshot 2022-01-26 at 10 44 34" src="https://user-images.githubusercontent.com/85135210/151122884-39498189-1272-4188-b822-6226615bcc5a.png">

<img width="318" alt="Screenshot 2022-01-26 at 10 44 18" src="https://user-images.githubusercontent.com/85135210/151122932-a1c541fc-10de-46e4-a057-9fad06658d49.png">

**Comment by crynobone** [comment]
Not able to replicate the issue and I personally don't advise using `Date` for `datetime` cast attributes.
